src/pages/billing/List.js

```diff
@@ -53,9 +53,10 @@ const billingProjectNameValidator = existing => ({
   }
 })
 
-const noBillingMessage = div({ style: { fontSize: 20, margin: '2rem' } }, [
+const noBillingMessage = onClickCreateProject => div({ style: { fontSize: 20, margin: '2rem' } }, [
   div([
-    'To get started, click the plus button to ', span({ style: { fontWeight: 600 } }, ['create a Billing Project'])
+    'To get started, ',
+    h(Link, { onClick: onClickCreateProject }, ['click here to ', span({ style: { fontWeight: 600 } }, ['create a Billing Project'])])
```

Comment:
Alternative: make just the `click here` text a link and leave the rest of the styling alone. Thoughts?

```suggestion
    h(Link, { onClick: onClickCreateProject }, ['click here']),
    ' to ', span({ style: { fontWeight: 600 } }, ['create a Billing Project'])
```

Reply:
Generally I think it's better for accessibility for links to be self-describing if possible, so the current form seems better. Maybe we don't need the bold font anymore though?

Reply:
Right. Thanks for the reminder of that accessibility rule. I agree that the bold can probably go.